src/Components/Nav/nav.css

```diff
@@ -1,5 +1,15 @@
 nav {
```

Comment:
make nav responsive for mobile

Perhaps a media query that changes flex-direction to column?